src/azul/service/manifest_controller.py

```diff
@@ -109,28 +113,38 @@ def get_manifest_async(self,
             except KeyError:
                 object_key, manifest = self.service.get_cached_manifest(format_=format_,
                                                                         catalog=catalog,
-                                                                        filters=filters)
+                                                                        filters=filters,
+                                                                        auth=authentication)
                 if manifest is None:
                     assert object_key is not None
                     partition = ManifestPartition.first()
                     state = {
                         'format_': format_.value,
                         'catalog': catalog,
                         'filters': filters.to_json(),
+                        'auth': None if authentication is None else authentication.to_json(),
```

Comment:
```suggestion
                        'authentication': None if authentication is None else authentication.to_json(),
```

Reply:
Alternatively, I'd be happy to use `authn` for variable names and dictionary keys but then we'd have to add a refactoring commit that changes the precedent, which I believe is `authentication` currently.